packages/web/src/components/gcds-radios/stories/gcds-radios.stories.tsx

```diff
@@ -0,0 +1,312 @@
+        {
+          id: string;
+          label: string;
+          value: string;
+          hint?: string;
+          checked?: boolean;
+          required?: boolean;
+          disabled?: boolean;
+        }
+        string | object[]`,
+        },
+        defaultValue: { summary: '-' },
+      },
+      type: {
+        required: true,
+      },
+    },
+    disabled: {
+      control: { type: 'select' },
+      options: [false, true],
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    errorMessage: {
+      name: 'error-message',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    hint: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    legend: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+      type: {
+        required: true,
+      },
+    },
+    required: {
+      control: { type: 'select' },
+      options: [false, true],
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    value: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    ...langProp,
+    ...validatorProps,
+
+    // Events
+    gcdsChange: {
+      action: 'RadioChange',
+      ...eventProp,
+    },
+    gcdsFocus: {
+      action: 'focus',
+      ...eventProp,
+    },
+    gcdsBlur: {
+      action: 'blur',
+      ...eventProp,
+    },
+  },
+};
+
+const Template = args =>
+  `
+<!-- Web component code (HTML, Angular, Vue) -->
+<gcds-radios
+  name="${args.name}"
+  legend="${args.legend}"
+  options='${args.options}'
+  ${args.hint ? `hint="${args.hint}"` : null}
+  ${args.errorMessage ? `error-message="${args.errorMessage}"` : null}
+  ${args.required ? `required` : null}
+  ${args.disabled ? `disabled` : null}
+  ${args.value ? `value="${args.value}"` : null}
+  ${args.validateOn != 'blur' ? `validate-on="${args.validateOn}"` : null}
+  ${args.lang != 'en' ? `lang="${args.lang}"` : null}
+>
+</gcds-radios>
+
+<!-- React code -->
+<GcdsRadios
+  name="${args.name}"
+  legend="${args.legend}"
+  options='${args.options}'
+  ${args.hint ? `hint="${args.hint}"` : null}
+  ${args.errorMessage ? `errorMessage="${args.errorMessage}"` : null}
+  ${args.required ? `required` : null}
+  ${args.disabled ? `disabled` : null}
+  ${args.value ? `value="${args.value}"` : null}
+  ${args.validateOn != 'blur' ? `validate-on="${args.validateOn}"` : null}
+  ${args.lang != 'en' ? `lang="${args.lang}"` : null}
+>
+</GcdsRadios>
+`.replace(/\s\snull\n/g, '');
+
+const TemplatePlayground = args =>
+  `
+<!-- Web component code (Angular, Vue) -->
+<gcds-radios
+  name="${args.name}"
```

Comment:
Seems like the legend is missing here:

```suggestion
  name="${args.name}"
  legend="${args.legend}"
```

Reply:
I added the rest of the properties as well, it was weird it only had name and options.